_inc/client/plans/plan-body.jsx

```diff
@@ -78,7 +79,57 @@ const PlanBody = React.createClass( {
 		const planClass = 'dev' !== this.props.plan
 			? getPlanClass( this.props.plan )
 			: 'dev';
-		const premiumThemesActive = includes( this.props.activeFeatures, FEATURE_UNLIMITED_PREMIUM_THEMES );
+		const premiumThemesActive = includes( this.props.activeFeatures, FEATURE_UNLIMITED_PREMIUM_THEMES ),
+			rewindActive = 'active' === get( this.props.rewindStatus, [ 'state' ], false );
```

Comment:
Calypso's standard is to add const in new lines. That prevents this bad indentation and also showing lines like this one in diffs.